pkgs/applications/editors/emacs/generic.nix

```diff
@@ -146,7 +146,9 @@ let emacs = (if withMacport then llvmPackages_6.stdenv else stdenv).mkDerivation
     ++ lib.optionals withX
       [ xlibsWrapper libXaw Xaw3d libXpm libpng libjpeg giflib libtiff libXft
         gconf cairo ]
-    ++ lib.optionals (withX || withNS) [ librsvg ]
+    ++ lib.optionals withPgtk
+      [ libXpm libpng libjpeg giflib libtiff ]
```

Comment:
What is libXpm for? That sounds like an x11-specific library, no?

Reply:
I understand [it's for parsing the XPM image format](https://gitlab.freedesktop.org/xorg/lib/libxpm). NixOS packages it under `xorg.`, but it's not X-specific.